workspaces/sonarqube/.changeset/sixty-readers-work.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage-community/plugin-sonarqube-backend': patch
+'@backstage-community/plugin-sonarqube-react': patch
+'@backstage-community/plugin-sonarqube': patch
```

Comment:
Can you change this to `minor`, please? If we need to back port a patch this gives us the ability to do so.